external/js/cothority/lib/protobuf/build/build_proto.js

```diff
@@ -0,0 +1,19 @@
+const protobuf = require('protobufjs');
```

Comment:
Are these auto-generated? Can you add some docs on how to generate them?

Reply:
So the `protobufjs` is an external protobuf library. From that and the list of protobuf definitions we generate a JSON representation, that gets embedded in the general library. 
There is an instruction on how to build the JSON representation (if you wanna add a new protobuf definition) in the README.